nixos/modules/services/hardware/asusctl.nix

```diff
@@ -0,0 +1,482 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.asusctl;
+
+  localAsusdConfig = {
+    bat_charge_limit = 100;
+    panel_od = false;
+  };
+
+  localAuraConfig = {
+    brightness = "Med";
```

Comment:
From where did you get this? How did you generate it? Should we move it into an extra file?

Reply:
Found it from the first start of the binary, I removed it as it is better to rely on the default of the binary.